src/components/dls/Button/Button.tsx

```diff
@@ -29,6 +29,7 @@ export enum ButtonType {
   Success = 'success',
   Error = 'error',
   Warning = 'warning',
+  Inverse = 'inverse',
```

Comment:
Can we add an example to Storybook?

Reply:
We currently don't add an example for each variant, instead we add it as a prop and allow you to play with different values.